assets/app/scripts/filters/resources.js

```diff
@@ -926,11 +926,24 @@ angular.module('openshiftConsole')
       if (!route.status.ingress) {
         return "Pending";
       }
-      var admittedCondition = routeIngressConditionFilter(route, 'Admitted');
-      if (!admittedCondition) {
+
+      var admittedConditions = [];
+      angular.forEach(route.status.ingress, function(ingress) {
+        var admitted = routeIngressConditionFilter(ingress, 'Admitted');
+        if (admitted) {
+          admittedConditions.push(admitted);
+        }
+      });
+
+      if (!admittedConditions.length) {
         return "Unknown";
       }
-      return admittedCondition.status === "True" ? "Admitted" : "Rejected";
+
+      if (_.some(admittedConditions, { status: "True" })) {
```

Comment:
Yeah I agree, when I originally added it I was using it somewhere else but
there is no point in it being its own filter now.
On Mar 1, 2016 4:11 PM, "Sam Padgett" <notifications@github.com> wrote:

> In assets/app/scripts/filters/resources.js
> <https://github.com/openshift/origin/pull/7725#discussion_r54636170>:
>
> >          return "Unknown";
> >        }
> > -      return admittedCondition.status === "True" ? "Admitted" : "Rejected";
> > +
> > +      if (_.some(admittedConditions, { status: "True" })) {
>
> I think the problem is we're trying to show multiple status as one value.
> We're only using this for preferred display routes. I suggest we just move
> the logic to route service.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7725/files#r54636170>.
>